app/code/Magento/Captcha/etc/frontend/di.xml

```diff
@@ -37,4 +37,18 @@
     <type name="Magento\Sales\Api\OrderManagementInterface">
         <plugin name="reset_payment_attempts_after_order_is_placed_plugin" type="Magento\Captcha\Plugin\ResetPaymentAttemptsAfterOrderIsPlacedPlugin"/>
     </type>
+    <type name="Magento\Checkout\Block\Onepage">
+        <arguments>
+            <argument name="layoutProcessors" xsi:type="array">
+                <item name="checkout_processor" xsi:type="object">Magento\Captcha\Block\CheckoutLayoutProcessor</item>
```

Comment:
In terms of checkout onepage block, it's more related to captaha, so let's change the name here
```suggestion
                <item name="captcha_processor" xsi:type="object">Magento\Captcha\Block\CheckoutLayoutProcessor</item>
```